armi/operators/settingsValidation.py

```diff
@@ -219,6 +226,38 @@ def run(self, cs=None):
                 )
             runLog.debug("{} has finished querying.".format(self.__class__.__name__))
 
+        # decide what to do with original settings file
+        if correctionsMade:
+            oldFilePath = "{}_old.yaml".format(self.cs.path.split(".yaml")[0])
+            if self._csRelativePathExists(oldFilePath):
+                overwrite_file = prompt(
+                    'INSPECTOR: At least one corrective query has been resolved and an "_old.yaml" settings file {} already exists.'.format(
+                        oldFilePath
+                    ),
+                    'Overwrite "_old.yaml" settings file to preserve originally submitted settings file?',
+                    "YES_NO",
+                    "NO_DEFAULT",
+                    "CANCEL",
+                )
+                if overwrite_file:
+                    os.replace(tempFilePath, oldFilePath)
+            else:
+                preserve_settings = prompt(
+                    "INSPECTOR: At least one corrective query has been resolved.",
+                    "Preserve originally submitted settings in: {}?".format(
+                        oldFilePath
+                    ),
+                    "YES_NO",
+                    "NO_DEFAULT",
+                    "CANCEL",
+                )
```

Comment:
I recommend always preserving and deleting the prompt. its another yes/no that someone will be asked every time... just make it  ( can give a message it is being made) and if the user doesn't want it, they can delete manually which is probably easier than another yes/no.